src/navigation/navigation.js

```diff
@@ -48,11 +47,7 @@ const component = {};
   *
   */
 
-component.template = `
-<bd-accordion single-section-only="true">
-    <nav class="navigation" ng-transclude></nav>
-</bd-accordion>
-`;
+component.template = '<bd-accordion single-section-only="true"><nav class="navigation" ng-transclude></nav></bd-accordion>';
```

Comment:
this change is annoying, but it stops some of the warnings when you build

Reply:
why is this an issue here and not in action-input.js?

Reply:
https://media.giphy.com/media/RRqbcP3ads9AQ/giphy.gif
🤷‍♀️